src/Numerics/Mesh/Elements.jl

```diff
@@ -9,8 +9,12 @@ Gauss-Legendre-Lobatto quadrature rule of type `T`
 
 """
 function lglpoints(::Type{T}, N::Integer) where {T <: AbstractFloat}
-    @assert N ≥ 1
-    GaussQuadrature.legendre(T, N + 1, GaussQuadrature.both)
+    @assert N ≥ 0
+    if N == 0
+        return (Array([T(0),] ), Array([T(2),]))
+    else
+        GaussQuadrature.legendre(T, N + 1, GaussQuadrature.both)
+    end
```

Comment:
LGL points always include end points so that change seems wrong, mathematically speaking. For `N = 0` you  probably want to use `glpoints` instead.

Reply:
I made the recommended revisions.